xla/hlo/ir/hlo_opcode.h

```diff
@@ -75,6 +75,9 @@ namespace xla {
   V(kCholesky, "cholesky", 1)                                                  \
   V(kClamp, "clamp", 3)                                                        \
   V(kClz, "count-leading-zeros", 1)                                            \
+  V(kCollectiveBroadcast, "collective-broadcast", kHloOpcodeIsVariadic)            \
+  V(kCollectiveBroadcastStart, "collective-broadcast-start", kHloOpcodeIsVariadic) \
```

Comment:
I think going forward, we want to discourage addition of specific start/done ops and use the generic async-start/done ops to support this. We do that for async reduce-scatter and all-to-all for example. Please see https://github.com/openxla/xla/blob/main/docs/async_ops.md

Reply:
Aw ok nice, yeah that split was a pain to deal with. I'll update the PR to only use async-start/done.

Reply:
I've removed the kCollectiveBroadcast[Start,Done]